packages/cli/src/commands/ldap/reset.ts

```diff
@@ -110,7 +111,7 @@ export class Reset extends BaseCommand {
 		}
 
 		for (const credential of ownedCredentials) {
-			await Container.get(CredentialsService).delete(credential);
+			await Container.get(CredentialsRepository).remove(credential);
```

Comment:
this change means that we now do not run the `credentials.delete` hook.
I don't know if that is what we want or not. Just want to confirm that this change is intentional.

Reply:
That wasn't intentional. 

I changed it back to calling `CredentialsService.delete` and passing the owner. That way it's in line with deleting the workflow above it:

https://github.com/n8n-io/n8n/blob/09115e016b0289d4cd64bd50ffeef9794e39110c/packages/cli/src/commands/ldap/reset.ts#L108-L114